onnx/test/reference_evaluator_ml_test.py

```diff
@@ -829,22 +831,33 @@ def _get_test_tree_ensemble_regressor(
         return onx
 
     @unittest.skipIf(not ONNX_ML, reason="onnx not compiled with ai.onnx.ml")
-    def test_tree_ensemble_regressor(self):
+    @parameterized.expand(
+        [
+            (f"{agg}_{base_value}", base_value, agg)
+            for base_value in (None, [1.0])
```

Comment:
I would use `itertools.product` for this. A double for loop in list comprehension can be confusing: https://google.github.io/styleguide/pyguide.html#274-decision

Reply:
I don't know how to do this with itertools.product because of the name variable at the beginning. I do want to keep the name variable, since I think that is the whole point of the parameterization - that is to more clearly see any failures directly from the test name.